scipy/stats/_multivariate.py

```diff
@@ -34,14 +34,14 @@
 
 
 _doc_random_state = """\
-random_state : {None, int, np.random.RandomState, np.random.Generator}, optional
+seed : {None, int, np.random.RandomState, np.random.Generator}, optional
     Used for drawing random variates.
-    If `seed` is `None` the `~np.random.RandomState` singleton is used.
+    If `seed` is `None`, the `~np.random.RandomState` singleton is used.
     If `seed` is an int, a new ``RandomState`` instance is used, seeded
     with seed.
     If `seed` is already a ``RandomState`` or ``Generator`` instance,
     then that object is used.
-    Default is None.
+    Default is `None`.
```

Comment:
The problem is that when the class is callable to create a frozen distribution, the keyword parameter is `seed`, whereas the `rvs` method parameter is `random_state`. My opinion is that the section at the top of the class documentation, e.g.
![image](https://user-images.githubusercontent.com/6570539/99885015-e63a8780-2be6-11eb-87ba-2c59352c83ae.png)

should define only how the class should be called; it should not be responsible for defining the parameters used in the methods.

Reply:
> In the meantime, maybe a compromise that we can implement without too much trouble is to have separate parameters sections, e.g. "Class Parameters" and "Method Parameters". "Method Parameters" would not repeat the definitions of any parameters mentioned in "Class Parameters", it would just define the ones like x and random_state that are only used for the methods and not used when instantiating a frozen distribution

That's a great idea, thanks! I will check whether sections like `Class Parameters` and `Method Parameters` are valid in `numpydoc` and `scipy-sphinx-theme` and push the changes.